### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by TetianaLykholot**

### CODE REVIEW

File: src/main/java/Airport.java [hunk 1]
```diff
@@ -27,28 +25,19 @@ public List<MilitaryPlane> getMilitaryPlanes() {
         for (Plane plane : planes) {
             if (plane instanceof MilitaryPlane) {
                 militaryPlanes.add((MilitaryPlane) plane);
```
Comment (Stasnazarenko) [inline src/main/java/Airport.java]: please re-write with the stream/foreach from java 8

File: src/main/java/Planes/Plane.java [hunk 2]
```diff
@@ -19,19 +21,18 @@ public String getModel() {
         return model;
     }
 
-    public int getMS() {
+    public int getMaxSpeed() {
         return maxSpeed;
     }
 
-    public int Get_Max_Flight_Distance() {
+    public int GetMaxFlightDistance() {
```
Comment (Stasnazarenko) [inline src/main/java/Planes/Plane.java]: import lombok.Getter;
import lombok.Setter;
https://www.baeldung.com/intro-to-project-lombok

File: src/main/java/Airport.java (hunk 1 above)
Comment (TetianaLykholot) [reply src/main/java/Airport.java]: done

File: src/main/java/Planes/Plane.java (hunk 2 above)
Comment (TetianaLykholot) [reply src/main/java/Planes/Plane.java]: done

**Pull request merged**